superset/dashboards/commands/importers/v0.py

```diff
@@ -324,7 +325,7 @@ def run(self) -> None:
         self.validate()
 
         for file_name, content in self.contents.items():
-            logger.info(f"Importing dashboard from file {file_name}")
+            logger.info("Importing dashboard from file %s", file_name)
```

Comment:
can we keep fstrings here?

Reply:
@hughhhh when logging pylint actually [recommends string interpolation](https://github.com/PyCQA/pylint/issues/1788), since it's faster (the string is interpolated only if the log level is being recorded) and safer (if the string interpolation fails something is still logged). I changed it here because pylint was complaining.